adapter/stdioLogger/stdioLogger.go

```diff
@@ -81,20 +79,13 @@ func (a *adapter) NewAspect(env aspect.Env, cfg proto.Message) (logger.Aspect, e
 		w = os.Stdout
 	}
 
-	name := "istio_log"
-	if c.LogName != "" {
-		name = c.LogName
-	}
-
 	pFmt := textFmt
 	if c.PayloadFormat == config.Params_STRUCTURED {
 		pFmt = structFmt
 	}
 
 	return &aspectImpl{
```

Comment:
I have a different preference. But I also have a way to work around this issue that I think may satisfy both of us.

OK with leaving it as is for now, and dealing with the fallout during the great re-org next week? I'll draft an example of my idea in time for that discussion.